.github/workflows/pythonbuild.yml

```diff
@@ -65,6 +91,10 @@ jobs:
           mysql -e 'create database IF NOT EXISTS luigi_test;' -uroot -proot || true
           mysql -e 'create user 'travis'@'localhost';' -uroot -proot || true
           mysql -e 'grant all privileges ON *.* TO 'travis'@'localhost';' -uroot -proot || true
+      - name: Create PSQL database
+        if: ${{ contains(matrix.tox-env, 'postgres') }} 
+        run: |
+          PGPASSWORD=postgres psql -h localhost -p 5432 -c 'create database spotify;' -U postgres
```

Comment:
https://github.com/actions/example-services/blob/5ba52cd792abf85235f6f446e56ea428519c60ec/.github/workflows/postgres-service.yml#L15-L18

Since there is no container involved in the builds, postgres is available through localhost